strato

```diff
@@ -205,25 +220,48 @@ else
 fi
 export sslCertFileType=${sslCertFileType:-crt}
 export NODE_NAME=${NODE_NAME:-$NODE_HOST}
+export authBasic=${authBasic:-true}
+export uiPassword=${uiPassword:-}
+export STRATO_GS_MODE=${mode}
+export SMD_MODE=${SMD_MODE:-enterprise}
+
+# BEGIN # Vars bloc for backwards-compatibility with STRATO 4.2 and older
 export BLOC_URL=${BLOC_URL:-${http_protocol}://$NODE_HOST/bloc/v2.2}
 export BLOC_DOC_URL=${BLOC_DOC_URL:-${http_protocol}://$NODE_HOST/docs/?url=/bloc/v2.2/swagger.json}
 export STRATO_URL=${STRATO_URL:-${http_protocol}://$NODE_HOST/strato-api/eth/v1.2}
 export STRATO_DOC_URL=${STRATO_DOC_URL:-${http_protocol}://$NODE_HOST/docs/?url=/strato-api/eth/v1.2/swagger.json}
 export CIRRUS_URL=${CIRRUS_URL:-${http_protocol}://$NODE_HOST/cirrus/search}
 export APEX_URL=${APEX_URL:-${http_protocol}://$NODE_HOST/apex-api}
-export authBasic=${authBasic:-true}
-export uiPassword=${uiPassword:-}
-export STRATO_GS_MODE=${mode}
-export SMD_MODE=${SMD_MODE:-enterprise}
+# ENDOF # Vars bloc for backwards-compatibility with STRATO 4.2 and older
 
 if [ "$SMD_MODE" = "enterprise" ] && [ -n "${EXT_STORAGE_S3_BUCKET}" ]; then
   if [[ -z ${EXT_STORAGE_S3_ACCESS_KEY_ID} || -z ${EXT_STORAGE_S3_SECRET_ACCESS_KEY} ]]; then
     echo -e "${Red}The external storage S3 bucket name is provided but one of the credentials is empty. Expected all or none of [EXT_STORAGE_S3_BUCKET, EXT_STORAGE_S3_ACCESS_KEY_ID, EXT_STORAGE_S3_SECRET_ACCESS_KEY]. Exit.${NC}"
     exit 6
+  fi
+fi
+
+# Process old OAUTH var names - for backwards-compatibility
+if [ "$OAUTH_JWT_VALIDATION_ENABLED" = "true" ]; then
+  if [ "$OAUTH_STRATO42_FALLBACK" = "true" ]; then
+    if [ -z ${OAUTH_JWT_VALIDATION_DISCOVERY_URL} ]; then
+      echo -e "${Red}OAUTH_JWT_VALIDATION_DISCOVERY_URL is required for OAUTH_JWT_VALIDATION_ENABLED mode with OAUTH_STRATO42_FALLBACK=true"
+      echo -e "For additional help see './strato --help'${NC}"
+      exit 14
+    fi
   else
-    export EXT_STORAGE_S3_BUCKET=${EXT_STORAGE_S3_BUCKET}
-    export EXT_STORAGE_S3_ACCESS_KEY_ID=${EXT_STORAGE_S3_ACCESS_KEY_ID}
-    export EXT_STORAGE_S3_SECRET_ACCESS_KEY=${EXT_STORAGE_S3_SECRET_ACCESS_KEY}
+    echo -e "${Red}OAUTH_JWT_VALIDATION_ENABLED and OAUTH_JWT_VALIDATION_DISCOVERY_URL variables are deprecated in STRATO v4.3+"
+    echo -e "For compatibility with STRATO v4.2, please use the 'OAUTH_STRATO42_FALLBACK=true' mode"
+    echo -e "For additional help see './strato --help'${NC}"
+    exit 11
+  fi
+fi
+
+if [ ${OAUTH_ENABLED} = true ]; then
```

Comment:
`if [[ ${OAUTH_ENABLED = true ]];` or `if [ ${OAUTH_ENABLED:-false} = true ];` would be preferable. I get `./strato.sh: line 260: [: =: unary operator expected` when it is unset